aspnetcore/security/how-to-choose-identity-solution.md

```diff
@@ -22,38 +22,64 @@ ASP.NET Core ships with a built-in authentication provider: [ASP.NET Core Identi
+A common reason a token server is required is for applications that exposes web-based APIs that are consumed by other apps. For exposed web-based APIs, client UIs such as Single Page Applications (SPA), mobile clients, and desktop clients are considered to be part of the same app. SPA examples include Angular, React, and Blazor WebAssembly. If apps other than your web app or any client UIs must make a secure API call to your app, you'll likely want to use tokens.  If you only have client UIs, ASP.NET Core Identity provides the option to acquire a token during authentication. The authentication token:
 
-There are a few more considerations to help you decide which solution is right for you.  
+* Can be used by mobile and desktop clients.
+* Isn't suitable for managing access from external apps.
 
-## Consider disconnected scenarios
+Another reason a token server is required is for sharing sign-ins with other apps. Commonly referred to as *single sign on*, this feature enables users to:
 
-Many solutions, such as [Microsoft Entra ID](/azure/active-directory/fundamentals/active-directory-whatis), are cloud-based and require an Internet connection to work. If your environment does not allow Internet connectivity, you will not be able to use the service. Consider the environment your app will run in. If you must support scenarios where either your app cannot access the Internet or must still function on the local network even if the Internet is disconnected, be sure to choose a solution that allows you to either install and run the service on your machines, or run it locally as a container.
+* Sign in once with a web app's form.
+* Use the resulting credentials to authenticate with other apps without having to sign-in again or choose a different password.
 
-## Decide where user data such as logins will be stored
+A token server is typically preferred to provide a secure and scalable solution for single sign on.
 
-Another important factor to consider is where the user login data will be stored. Many developers choose external, cloud-based services like Microsoft Entra ID to manage identity because the service provider takes on the responsibilities of securely storing data, keeping the software up to date with the latest security patches and releases, and complying with privacy regulations. Others prefer to store data on their own servers due to regulations, compliance, policy, and other reasons.
+For apps that don't have either of these requirements, the simplest way to quickly secure an app is to use the built-in ASP.NET Core Identity provider. Otherwise, a token server provided by a third-party identity management solution is needed. Token servers are available as:
```

Comment:
```suggestion
For apps that don't exposes web-based APIs that are consumed by other apps and don't share logins with other apps, the simplest way to quickly secure an app is to use the built-in ASP.NET Core Identity provider. Otherwise, a token server provided by a third-party identity management solution is needed. Token servers are available as:
```